src/mainboard/protectli/vault_adl_p/devicetree.cb

```diff
@@ -0,0 +1,349 @@
+						register "use_custom_pld" = "true"
+						register "custom_pld" = "ACPI_PLD_TYPE_A(FRONT, RIGHT, ACPI_PLD_GROUP(1, 2))"
+						register "custom_pld.vertical_position" = "PLD_VERTICAL_POSITION_UPPER"
+						device ref usb2_port2 on end
+					end
+					chip drivers/usb/acpi
+						register "desc" = ""USB2 5G (MKB_4G1)""
+						register "type" = "UPC_TYPE_INTERNAL"
+						device ref usb2_port3 on end
+					end
+					chip drivers/usb/acpi
+						register "desc" = ""USB3 Type-A Internal""
+						register "type" = "UPC_TYPE_INTERNAL"
+						device ref usb2_port4 on end
+					end
+					chip drivers/usb/acpi
+						register "desc" = ""USB2 Type-A Port 4 (USB2)""
+						register "type" = "UPC_TYPE_A"
+						register "use_custom_pld" = "true"
+						register "custom_pld" = "ACPI_PLD_TYPE_A(FRONT, LEFT, ACPI_PLD_GROUP(3, 1))"
+						register "custom_pld.vertical_position" = "PLD_VERTICAL_POSITION_LOWER"
+						device ref usb2_port6 on end
+					end
+					chip drivers/usb/acpi
+						register "desc" = ""USB2 Type-C Port""
+						register "type" = "UPC_TYPE_C_USB2_SS_SWITCH"
+						register "use_custom_pld" = "true"
+						register "custom_pld" = "ACPI_PLD_TYPE_C(FRONT, RIGHT, ACPI_PLD_GROUP(2, 1))"
+						device ref usb2_port8 on end
+					end
+					chip drivers/usb/acpi
+						register "desc" = ""USB2 Type-A Port 3 (USB2)""
+						register "type" = "UPC_TYPE_A"
+						register "use_custom_pld" = "true"
+						register "custom_pld" = "ACPI_PLD_TYPE_A(FRONT, LEFT, ACPI_PLD_GROUP(3, 2))"
+						register "custom_pld.vertical_position" = "PLD_VERTICAL_POSITION_UPPER"
+						device ref usb2_port9 on end
+					end
+					chip drivers/usb/acpi
+						register "desc" = ""USB2 WiFi/Bluetooth""
+						register "type" = "UPC_TYPE_INTERNAL"
+						device ref usb2_port10 on end
+					end
+				end
+			end
+		end
+
+
+		device ref i2c1 on
+			register "serial_io_i2c_mode[PchSerialIoIndexI2C1]" = "PchSerialIoPci"
+			chip drivers/i2c/generic
+				register "hid" = ""INT3515""
+				register "uid" = "1"
+				register "name" = ""PD01""
+				register "desc" = ""TPS65994 USB-C PD Controller""
+				register "irq_gpio" = "ACPI_GPIO_IRQ_LEVEL_LOW(GPP_E2)"
+				device i2c 23 on end
+			end
+		end
+		device ref heci1 on end
+		device ref sata on
+			register "sata_salp_support" = "1"
+
+			register "sata_ports_enable" = "{
+				[0] = 1,
+				[1] = 1,
+			}"
+
+			register "sata_ports_hotplug" = "{
+				[0] = 1,
+				[1] = 1,
+			}"
+		end
+
+		#x4 link to Intel XL710 2xSFP
+		device ref pcie_rp1 on
+			register "pch_pcie_rp[PCH_RP(1)]" = "{
+				.flags = PCIE_RP_LTR | PCIE_RP_CLK_SRC_UNUSED |
+					 PCIE_RP_CLK_REQ_UNUSED | PCIE_RP_BUILT_IN,
+				.pcie_rp_aspm = ASPM_L0S,
+			}"
+			register "pcie_clk_config_flag[1]" = "PCIE_CLK_FREE_RUNNING"
+		end
+		# RP5-RP8 i226 LAN
+		device ref pcie_rp5 on
+			register "pch_pcie_rp[PCH_RP(5)]" = "{
+				.flags = PCIE_RP_LTR | PCIE_RP_CLK_SRC_UNUSED |
+					 PCIE_RP_CLK_REQ_UNUSED | PCIE_RP_BUILT_IN,
```

Comment:
We might reevaluate the CLK_REQ if this makes any sense: https://github.com/Dasharo/coreboot/pull/455#discussion_r1473439247

Reply:
https://github.com/Dasharo/coreboot/pull/455#discussion_r1474169263